### ISSUE TITLE
False negative with accessing super types private member from sub type

### ISSUE THREAD

**Issue by KotlinIsland** [python/mypy#9446]
Following code shows no errors from mypy but will fail at runtime.
```py
class A:
    def __a(self) -> None:
        ...


class B(A):
    def foo(self) -> None:
        self.__a()


B().foo()
```

**Comment by gvanrossum** [comment]
Thanks for the report -- this looks like a real bug. Fixing it would probably be fairly involved (we'd have to implement name mangling at the right level in the parser) and private members aren't that popular. So I'm classifying this as low priority.

**Comment by AlexWaygood** [comment]
Closing as a duplicate of #8267
